src/metabase/lib/hierarchy.cljc

```diff
@@ -7,7 +7,9 @@
 (defn derive
   "Like [[clojure.core/derive]], but affects [[hierarchy]] rather than the global hierarchy."
   [tag parent]
-  (swap! hierarchy clojure.core/derive tag parent))
+  (swap! hierarchy clojure.core/derive tag parent)
+  ;; for REPL convenience so we don't dump a lot of garbage
+  nil)
```

Comment:
Braden and I talked about doing this in Slack, it's a little annoying to use this function from the REPL and have it dump out the entire state of the the MLv2 `hierarchy`